rholang/src/test/scala/coop/rchain/rholang/interpreter/SubstituteTest.scala

```diff
@@ -188,3 +188,38 @@ class EvalSubSpec extends FlatSpec with Matchers {
     result should be(expected)
   }
 }
+
+class BundleSubSpec extends FlatSpec with Matchers {
+  "Bundle" should "only substitute body of expression" in {
```

Comment:
"Bundle" should "substitute within the body of the bundle."

Reply:
Fixed.